tests/test_vote_simple_dvt.py

```diff
@@ -0,0 +1,631 @@
+"""
+Tests for voting 23/01/2023
+
+"""
+
+import pytest
+from typing import List
+from scripts.vote_simple_dvt import start_vote
+from brownie import interface, ZERO_ADDRESS, reverts, web3, accounts, convert, network
+from tests.regression.test_permissions import active_aragon_roles, protocol_permissions
+from utils.test.deposits_helpers import fill_deposit_buffer
+from utils.test.event_validators.aragon import validate_app_update_event, validate_push_to_repo_event
+from utils.test.event_validators.common import validate_events_chain
+from utils.test.event_validators.staking_router import StakingModuleItem, validate_staking_module_added_event
+from utils.test.oracle_report_helpers import oracle_report
+from utils.test.extra_data import ExtraDataService
+from utils.test.oracle_report_helpers import oracle_report
+from utils.test.simple_dvt_helpers import (
+    fill_simple_dvt_ops,
+    fill_simple_dvt_ops_vetted_keys,
+    get_managers_address,
+    get_operator_address,
+)
+from utils.test.tx_tracing_helpers import *
+from utils.config import (
+    contracts,
+    LDO_HOLDER_ADDRESS_FOR_TESTS,
+    network_name,
+    SIMPLE_DVT,
+    SIMPLE_DVT_IMPL,
+    SIMPLE_DVT_ARAGON_APP_NAME,
+    SIMPLE_DVT_ARAGON_APP_ID,
+    SIMPLE_DVT_MODULE_STUCK_PENALTY_DELAY,
+    SIMPLE_DVT_MODULE_TARGET_SHARE_BP,
+    SIMPLE_DVT_MODULE_MODULE_FEE_BP,
+    SIMPLE_DVT_MODULE_TREASURY_FEE_BP,
+    SIMPLE_DVT_MODULE_ID,
+    SIMPLE_DVT_MODULE_NAME,
+    SIMPLE_DVT_MODULE_TYPE,
+    EASYTRACK_SIMPLE_DVT_TRUSTED_CALLER,
+    EASYTRACK_EVMSCRIPT_EXECUTOR,
+    EASYTRACK_SIMPLE_DVT_ADD_NODE_OPERATORS_FACTORY,
+    EASYTRACK_SIMPLE_DVT_ACTIVATE_NODE_OPERATORS_FACTORY,
+    EASYTRACK_SIMPLE_DVT_DEACTIVATE_NODE_OPERATORS_FACTORY,
+    EASYTRACK_SIMPLE_DVT_SET_VETTED_VALIDATORS_LIMITS_FACTORY,
+    EASYTRACK_SIMPLE_DVT_SET_NODE_OPERATOR_NAMES_FACTORY,
+    EASYTRACK_SIMPLE_DVT_SET_NODE_OPERATOR_REWARD_ADDRESSES_FACTORY,
+    EASYTRACK_SIMPLE_DVT_UPDATE_TARGET_VALIDATOR_LIMITS_FACTORY,
+    EASYTRACK_SIMPLE_DVT_CHANGE_NODE_OPERATOR_MANAGERS_FACTORY,
+    REPO_APP_ID,
```

Comment:
not sure if it's good to import the same addresses that were used in the voting script
(usually prefer hardcoding in the test)

Reply:
as discussed, this is not a problem at the moment since we are checking on-chain state against a hard-coded constants, not the attentiveness of the programmer)